test/avocado/testlib_avocado/seleniumlib.py

```diff
@@ -56,6 +58,9 @@ class SeleniumTest(Test):
     :avocado: disable
     """
     def setUp(self):
+        # ensure that setup phase passed, otherwise handle it in tierDown
```

Comment:
As you consistently misspell this: It's "tear down", not "tier" (also in commit message). :smiley: